projects/cm_mcu/Tasks.h

```diff
@@ -219,6 +232,10 @@ const char *gitVersion(void);
 #define ALM_STAT_FIREFLY_OVERTEMP 0x2
 #define ALM_STAT_FPGA_OVERTEMP    0x4
 #define ALM_STAT_DCDC_OVERTEMP    0x8
+// status register bits
+// FIXME : copy from temp
```

Comment:
What does this FIXME mean?

Reply:
I assume that status flags of a power voltage for warning have the same values as those for temperature